File: app/src/main/java/com/example/watering/investrecord/DBHelper/SpendCardDBHelper.java
package com.example.watering.investrecord.DBHelper;

import android.content.Context;

/**
 * Created by watering on 17. 10. 21.
 */

@SuppressWarnings("DefaultFileTemplate")
public class SpendCardDBHelper extends DBHelper {

    public SpendCardDBHelper(Context context) {
        super(context);

        TABLE_NAME = "tbl_spend_card";
        COLUMNS = new String [] {"spend_code TEXT UNIQUE","id_card INTEGER"};
    }
}

File: app/src/main/java/com/example/watering/investrecord/dbhelper/SpendCardDBHelper.kt
package com.example.watering.investrecord.dbhelper

import android.content.Context

/**
 * Created by watering on 17. 10. 21.
 */

class SpendCardDBHelper(context: Context) : DBHelper(context) {

    init {

        TABLE_NAME = "tbl_spend_card"
        COLUMNS = arrayOf("spend_code TEXT UNIQUE", "id_card INTEGER")
    }
}
